Training-DT/todo-vue
# Add todo list to firebase firestore

### Overview
- Create input form
- Add todo list data to firestore

### How to test
- Input text in input box and click button add todo
- check document in myTodo firestore database
